examples/integration/test/fixtures/bwb.xcodeproj/xcshareddata/xcschemes/WidgetExtension.xcscheme

```diff
@@ -39,6 +39,22 @@
                </EnvironmentBuildable>
             </ActionContent>
          </ExecutionAction>
+         <ExecutionAction
+            ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
+            <ActionContent
+               title = "Set Bazel Build Output Groups for iOSApp"
+               scriptText = "echo &quot;$BAZEL_LABEL,$BAZEL_TARGET_ID&quot; &gt;&gt; &quot;$SCHEME_TARGET_IDS_FILE&quot;&#10;">
```

Comment:
This added `ExecutionAction` is intentional, right? Just wondering since it'll, in theory, be appending to the same `$SCHEME_TARGET_IDS_FILE`.

Reply:
Correct. The purpose of the PR is to add these, so the additional values in `$SCHEME_TARGET_IDS_FILE` will be used by `calculate_output_groups.py` in the fallback case.